app/domain/user.ts

```diff
@@ -239,6 +239,13 @@ export const setUserInFscNeuBeantragenProcess = async (
   });
 };
 
+export const saveFormData = async (id: string, formData: object) => {
+  return db.user.update({
+    where: { id },
+    data: { formData },
+  });
+};
+
```

Comment:
I assumed that the data would be stored in redis but this stores it in the database or am I mistaken?

Reply:
I tried Redis first and switched to Postgres afterwards. Both work. My line of thinking was "why not store the user's form data in the same place as the other user data? -> postgres". Also assuming that backup for postgres is already set up. When deleting an account, it's one place less to look at.

I guess, I would like to challenge the decision for redis we did in the past. Mostly because I cannot remember, what exactly we assumed back than. I vaguely remember we (also) talked about server-side sessions in Redis and maybe form data is a different use case.

Reply:
> I vaguely remember we (also) talked about server-side sessions in Redis and maybe form data is a different use case.

Yes, that is exactly where this came from. The question would be if the form data is session data or not. However, I see your points for Postgres. The only problem I see with using Postgres would be that with our current setup we would have a lot more db access which might slow down the site. Especially, if the site is under load. Perhaps we could run some load tests to check whether that is the case 🤔

Reply:
+1 for trying Postgres. In STL, form data was intended to be ephemeral and even there, we discussed storing it in Postgres. I believe we went with Redis because it makes auto-expiring entries trivial (unlike Postgres). Here, Postgres seems a better fit because the data is supposed to persist as long as there's a user.

+1 also for load testing. If we're feeling adventurous, we could try applying some (load) testing in production. Like this?
- Allow storing/reading data to/from both Cookie and DB at the same time.
- Use Unleash to turn on DB storage/reading for a percentage of users. (Still update the Cookie to allow turning off the feature without ramifications.)
- Slowly increase the percentage and monitor what happens to DB read/write load.